src/pages/Inventory.page.js

```diff
@@ -66,4 +72,31 @@ export class InventoryPage extends BaseSwagLabPage {
         let allNamesSorted = allNames.sort((a,b)=>b - a);
         expect(allNamesSorted).toEqual(allNames);
     }
+
+    async addRandomItemsToCart() {
+        let allNames = await this.page.locator('[data-test="inventory-item-name"]').allInnerTexts(); 
+        let selectedNames = [];
+        let allItesTogether= [];
+        for (let i = 0; i < 4; i++) {
+            const popedElement = this.popRandomElementFrom(allNames, selectedNames);
+            const price = await this.page.locator('[data-test="inventory-item-description"]').filter({ hasText: popedElement }).
+            locator('[data-test="inventory-item-price"]').innerText();
+            const description = await this.page.locator('[data-test="inventory-item-description"]').filter({ hasText: popedElement }).
+            locator('[data-test="inventory-item-desc"]').innerText();
+            allItesTogether.push({
+                name:popedElement,description: description, price:price
+            })
+            await this.page.locator('[data-test="inventory-item-description"]').filter({ hasText: popedElement }).getByRole('button', { name: "Add to cart" }).click();
+        }
+        return allItesTogether;
+    }
+
+    popRandomElementFrom = (a, newA) => {
```

Comment:
```suggestion
    popRandomElementFrom(a, newA) {
```

Reply:
for this function it's better to create a separate file utils.js and export it from there